src/main/java/com/bravo/user/validator/UserValidator.java

```diff
@@ -10,6 +10,12 @@
 @Component
 public class UserValidator extends CrudValidator {
 
+  public void validateName(String name){
+    if(ValidatorUtil.isInvalid(name)){
+      throw new BadRequestException("'name' is required");
```

Comment:
Your validation logic doesn't match your error message here. The user could be providing a "name", but still see this error message if the name provided doesn't match the RegExp used. That information should be included in the error message to reduce debugging time and user frustration.

Reply:
I agree, after I went back and added the additional validation for preventing numbers and characters, this message was no longer sufficient. Additionally I wanted to add that with my initial implementation of this method in order to follow the requirements on the trello board, I wanted to point out that since the value was a PathVariable, it could not be null or empty, because then it wouldn't map to the GET API.